### DISCUSSION TITLE
关于 bootstrap dns 的疑问

### DISCUSSION THREAD

**Discussion by zweie** [pymumu/smartdns#1470] [Q&A] [answered]
如果将一个 dns 指定为 bootstrap，那它还算在默认组里吗？还是说，设置了 `-bootstrap-dns`，等同于设置 `-group bootstrap-dns`？

**Comment by PikuZheng** [comment, marked as answer]
-bootstrap-dns 不会计入上游服务器组

**Reply by zweie** [reply]
明白了，谢谢
